azure-pipelines.yml

```diff
@@ -38,12 +34,17 @@ jobs:
     pool:
       name: 00-OSIManaged-Test
       demands: COMPUTERNAME -equals $(buildAgent)
-    steps:
-      - template: '/miscellaneous/build_templates/config.yml@templates'
+    variables:
+      - name: ClientId
+        value: $(AuthorizationCodeClientId)
+      - name: Username
+        value: $(AuthorizationUsername)
+      - name: Password
+        value: $(AuthorizationPassword)
+    steps:        
+      - template: '/miscellaneous/build_templates/appsettings.yml@templates'
         parameters:
-          fileOriginal: $(CredFile)
-          fileLibrary: $(CredFileLibrary)
-          filePath: /AuthorizationCodeFlow
+          secrets: 'TenantId, NamespaceId, AuthorizationUsername, AuthorizationPassword, AuthorizationCodeClientId'
```

Comment:
where is it pulling the tenantid and namespaceid from?

Reply:
This is pulled from the Azure Key Vault